people.md

```diff
@@ -2,87 +2,88 @@
-
-His research focuses on experimental methods, reconstruction algorithms, and software codes for simulation, analysis, and instrument design of TEM, STEM and 4DSTEM. He was awarded a US Department of Energy (DOE) Early Career award in 2018, and the [Burton Medal](https://microscopy.org/Society-Awards-Recipients#2022) from the [Microscopy Society of America](https://microscopy.org/) (MSA) in 2018. He has published over 200 peer-reviewed publications, and given over 100 invited talks and workshops around the world. Colin advocates for open science and his group develops open-source scientific software such as the [Prismatic STEM simulation](https://prism-em.com/) and [py4DSTEM analysis](https://github.com/py4dstem/py4DSTEM) codes. He is also the editor-in-chief for the interactive web-based scientific journal [Elemental Microscopy](https://www.elementalmicroscopy.com/).
+:::
 
+Colin received his BSc in Engineering Physics, and his PhD in Materials Engineering in [David Mitlin's](https://www.me.utexas.edu/people/faculty-directory/mitlin) group at the [University of Alberta](https://www.ualberta.ca/en/index.html) in Edmonton, Canada. He received an NSERC scholarship to pursue postdoctoral research working for [Velimir "Mimo" Radmilovic](https://www.sanu.ac.rs/en/member/radmilovic-velimir/) at Berkeley Lab and [Mark Asta](https://asta.mse.berkeley.edu/) at UC Berkeley. He then worked as a Project Scientist, Research Scientist, and Staff Scientist at the [National Center for Electron Microscopy](https://foundry.lbl.gov/about/facilities/the-national-center-for-electron-microscopy-ncem/) (NCEM), part of the [Molecular Foundry](https://foundry.lbl.gov/) user facility at [Lawrence Berkeley Lab](https://www.lbl.gov/).
 
+His research focuses on experimental methods, reconstruction algorithms, and software codes for simulation, analysis, and instrument design of TEM, STEM and 4DSTEM. He was awarded a US Department of Energy (DOE) Early Career award in 2018, and the [Burton Medal](https://microscopy.org/Society-Awards-Recipients#2022) from the [Microscopy Society of America](https://microscopy.org/) (MSA) in 2018. He has published over 200 peer-reviewed publications, and given over 100 invited talks and workshops around the world. Colin advocates for open science and his group develops open-source scientific software such as the [Prismatic STEM simulation](https://prism-em.com/) and [py4DSTEM analysis](https://github.com/py4dstem/py4DSTEM) codes. He is also the editor-in-chief for the interactive web-based scientific journal [Elemental Microscopy](https://www.elementalmicroscopy.com/).
 
 ## Postdoctoral Researchers
 
-### Stephanie Ribet
-
-```{image} /images/people/RibetSteph.jpg
-:alt: Steph
-:class: bg-primary
-:width: 200px
-:align: right
-```
-
+:::{person}
+:name: Stephanie Ribet
+:position: Postdoctoral Researcher
+:pronouns: she/her
```

Comment:
I have added a space for pronouns, obviously up to you if you want to use that and also add to the rest of the people on this page.